src/megaclient.cpp

```diff
@@ -4506,6 +4519,17 @@ void MegaClient::updatesc()
                     {
                         break;
                     }
+#ifdef ENABLE_SYNC
+                    if (!(*it)->isSyncable())
+                    {
+                        mUnsyncableNodes.insert((*it)->nodehandle);
+                        auto nodehandle = std::to_string((*it)->nodehandle);
```

Comment:
Can we be sure the dbid has been assigned at this point, since it's for a different table?   I think it would be better to keep them independent, eg. in case of total node tree reload?

Reply:
Will fix.